nvOC

```diff
@@ -210,16 +210,42 @@ Reboot()
    sudo su -c "shutdown -r now"
 }
 
+############################### Restart Miner ##################################
```

Comment:
We have a kill command in 0miner to prevent running 2 instance of miner 
So just running 0miner again will kill old miner and start new one

Ideas ?

Reply:
Some thing like this:
NVOC=$(pwd) bash ./0miner

Reply:
We can have 2 commands here nvoc_restart and miner_restart 
nvoc_restart will kill all and restart nvoc (same as before) , miner_restart just restart the miner

Reply:
You can safely remove that command from 0miner, since 2main already perform that check. Remember you always need to restart 3main from scratch if you don't know which auto-switching the user is using.

Reply:
Agree.
Then to do nvoc_restart  we should kill em all then kill 3main so it start them all again.
To do the miner_restart just kill 3main and it will re-up does needed and miner.

Reply:
The nvoc-restart provided by @sizzlephizzle is fine, just wait he fixes the launch command.

Reply:
I mentioned a possible solution in my review at line 238

Reply:
Is this what you mean:
gnome-terminal --window-with-profile=mining

Reply:
Either this one for new window or that one for new tab